### PR TITLE
feat: remove copilot plugin feature flag and remove standalone api plugin

### PR DESCRIPTION
**Pull request by anchenyi**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/30199601

https://github.com/user-attachments/assets/b412ee23-a475-48ec-a45c-b24785f97d7b

### CODE REVIEW

File: packages/fx-core/src/component/generator/apiSpec/generator.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ export class SpecGenerator extends DefaultTemplateGenerator {
     let templateName = "";
     if (
       (capability === CapabilityOptions.apiPlugin().id ||
```
Comment (yuqizhou77) [inline packages/fx-core/src/component/generator/apiSpec/generator.ts:129]: Can CapabilityOptions.apiPlugin() be deleted?

**Review by huimiu — DISMISSED** [review]

File: packages/fx-core/src/component/generator/apiSpec/generator.ts (hunk 1 above)
Comment (anchenyi) [reply packages/fx-core/src/component/generator/apiSpec/generator.ts:129]: Yes, I will create an another task to do this clean up. The current UI update is a little bit urgent.

**Review by yuqizhou77 — DISMISSED** [review]

File: packages/fx-core/src/question/constants.ts [hunk 2]
```diff
@@ -912,9 +895,7 @@ export class MeArchitectureOptions {
     return [
       MeArchitectureOptions.newApi(),
       MeArchitectureOptions.apiSpec(),
-      isCopilotExtensionEnabled()
-        ? MeArchitectureOptions.botPlugin()
-        : MeArchitectureOptions.botMe(),
```
[the remark this answers is not in the document]
Comment (anchenyi) [reply packages/fx-core/src/question/constants.ts:917]: Notified. Thanks for the information.

File: packages/vscode-extension/package.json [hunk 3]
```diff
@@ -214,45 +214,25 @@
           "when": "false",
           "visibility": "collapsed"
         },
-        {
-          "id": "teamsfx-empty-project",
```
[the remark this answers is not in the document]
Comment (anchenyi) [reply packages/vscode-extension/package.json:218]: Deleted the unused views and activations, thanks!

**Review by yuqizhou77 — APPROVED** [review]

File: packages/vscode-extension/package.json [hunk 4]
```diff
@@ -57,7 +57,6 @@
     "onCommand:workbench.action.debug.start",
     "workspaceContains:/.fx/**/*",
     "workspaceContains:teamsapp.yml",
-    "onView:teamsfx-empty-project",
```
Comment (tecton) [inline packages/vscode-extension/package.json:60]: This should be replaced by a default view for non-TTK project.

**Review by tecton — APPROVED** [review]

File: packages/vscode-extension/package.json (hunk 4 above)
Comment (anchenyi) [reply packages/vscode-extension/package.json]: Fixed. Thanks.

**Review by yuqizhou77 — APPROVED** [review]

**Review by huimiu — DISMISSED** [review]

**Review by jayzhang — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 5]
```diff
@@ -297,8 +297,7 @@
   "core.createProjectQuestion.projectType.bot.detail": "Conversational or informative chat experiences that can automate repetitive tasks",
   "core.createProjectQuestion.projectType.bot.label": "Bot",
   "core.createProjectQuestion.projectType.bot.title": "App Features Using a Bot",
-  "core.createProjectQuestion.projectType.messageExtension.detail": "Search and take actions from the text input box in Teams and Outlook",
-  "core.createProjectQuestion.projectType.messageExtension.copilotEnabled.detail": "Search or initiate actions from the message composing area of Teams, Outlook and Copilot",
+  "core.createProjectQuestion.projectType.messageExtension.agentEnabled.detail": "Search or initiate actions from the message composing area of Teams, Outlook and Agent",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:300]: Proposed: Search or initiate actions from the message input box in Teams, Outlook, and Agent.

**Review by yuqizhou77 — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json (hunk 5 above)
Comment (anchenyi) [reply packages/fx-core/resource/package.nls.json]: Updated. Thanks

**Review by supkasar — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**